subsystems/RollerIntakeBase.java

```diff
@@ -1,5 +1,70 @@
 package org.frc5587.lib.subsystems;
 
-public abstract class RollerIntakeBase {
-    
-}
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public abstract class RollerIntakeBase extends SubsystemBase {
+    protected IntakeConstants constants;
+
+    // 1 leader variable is utilized, while an array for followers is used for ease of use
+    protected SpeedController leader;
+    protected SpeedController[] followers;
+    protected SpeedControllerGroup motorGroup;
+
+    // Create intake constants for implementation in methods
+    public static class IntakeConstants {
+        public boolean inverted;
+        public int stallLimit;
+        public int freeLimit;
+        public double throttle;
+
+        public IntakeConstants(int leaderID, int followerIDs, boolean inverted, int stallLimit, int freeLimit, double throttle) {
+            this.inverted = inverted;
+            this.stallLimit = stallLimit;
+            this.freeLimit = freeLimit;
```

Comment:
You shouldn't need all these constants, you only use `throttle` and in that case, you can just pass it in, you don't need `IntakeConstants`